Project/QtCreator/qctools-lib/qctools-lib.pro

```diff
@@ -67,3 +67,6 @@ SOURCES = \
     $$SOURCES_PATH/Core/FFmpegVideoEncoder.cpp
 
 include($$SOURCES_PATH/ThirdParty/qblowfish/qblowfish.pri)
+
+RESOURCES += \
+    $$SOURCES_PATH/Core/resources.qrc
```

Comment:
There is already a file name `resources.qrc` (in `/Resources/`), consider using another name else it could lead to random weird behavior with `Q_INIT_RESOURCE(resources);` used later.